### PR TITLE
localized hindi docs homepage

### PR DESCRIPTION
**Pull request by mittalyashu**
This PR localise the content of `content/hi/docs/home` directory in Hindi language

Closes https://github.com/kubernetes/website/issues/13914

Old PR: https://github.com/kubernetes/website/pull/14069

### CODE REVIEW

**Comment by mittalyashu** [comment]
Should I localise "Cloud Native Computing Foundation" into Hindi language too?

File: content/hi/docs/home/_index.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+weight: 10
+hide_feedback: true
+menu:
+  main:
+    title: "प्रलेखन"
+    weight: 20
+    post: >
+      <p>कुबरनेट्स का उपयोग वैचारिक, ट्यूटोरियल और संदर्भ प्रलेखन के साथ करना सीखें। आप <a href="/editdocs/" data-auto-burger-exclude data-proofer-ignore>डॉक्स में योगदान</a> करने में सहायता भी कर सकते हैं!</p>
+description: >
+  कुबरनेट्स एक ओपन सोर्स container orchestration engine for automating deployment, scaling, and management of containerized applications. यह ओपन सोर्स प्रोजेक्ट को क्लाउड नेटिव कम्प्यूटिंग फाउंडेशन।
```
Comment (sftim) [inline content/hi/docs/home/_index.md:20]: > container orchestration engine for automating deployment, scaling, and management of containerized applications.

This looks like a _lot_ of English - is that how you intended it @mittalyashu ?

File: content/hi/docs/home/supported-doc-versions.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+title: उपलब्ध प्रलेखन संस्करण
+content_type: custom
+layout: supported-versions
+card:
+  name: के बारे में
+  weight: 10
+  title: उपलब्ध प्रलेखन संस्करण
+---
+
+इस वेबसाइट में कुबेरनेट्स के वर्तमान और पिछले 4 संस्करणों के प्रलेखन हैं।
```
Comment (sftim) [inline content/hi/docs/home/supported-doc-versions.md:11]: The rest of this page seems to be missing. What happened here?

**Review by sftim — COMMENTED** [review]
I'm afraid I don't speak Hindi, but even with my limited grasp of it, this PR does not seem right.

@mittalyashu would you have time to revise it further?

**Comment by mittalyashu** [comment]
> I'm afraid I don't speak Hindi, but even with my limited grasp of it, this PR does not seem right.

No, I understand. I just wanted to ask if the Kubernetes docs guidelines says to keep "Cloud Native Computing Foundation" in English or localise it.

> @mittalyashu would you have time to revise it further?

I did re-creating the PR by comparing it with English one. May I know which part require the changes?

File: content/hi/docs/home/supported-doc-versions.md (hunk 2 above)
Comment (sftim) [reply content/hi/docs/home/supported-doc-versions.md:11]: Ah, no, that does look right (I forgot we'd automated it).

**Comment by sftim** [comment]
As the name of the organisation, it makes sense to me to keep Cloud Native Computing Foundation as-is.

File: content/hi/docs/home/supported-doc-versions.md (hunk 2 above)
Comment (mittalyashu) [reply content/hi/docs/home/supported-doc-versions.md:11]: Should I mark this conversation as "Resolved"?

File: content/hi/docs/home/_index.md (hunk 1 above)
Comment (mittalyashu) [reply content/hi/docs/home/_index.md:20]: I wanted to ask if I should localise these words too.

If I remember correctly, last time also we have left few words in english. Wanted to make sure how to proceed further?

Comment (sftim) [reply content/hi/docs/home/_index.md:20]: @kubernetes/sig-docs-hi-reviews thoughts?

File: content/hi/docs/home/_index.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+  description: ब्राउज़ टर्मिनोलॉजी, कमांड लाइन सिंटैक्स, API रिसोर्स टाइप्स, और प्रलेखन।
+  button: संदर्भ देखें
+  button_path: /docs/reference
+- name: contribute
+  title: प्रलेखन में योगदान करें
+  description: कोई भी व्यक्ति योगदान दे सकता है, चाहे आप इस प्रोजेक्ट के लिए नए हों या आप लंबे समय से हैं।
+  button: प्रलेखन में योगदान करें
+  button_path: /docs/contribute
+- name: release-notes
+  title: रिलीज नोट्स
```
Comment (sayanchowdhury) [inline content/hi/docs/home/_index.md:60]: `रिलीज` -> `रिलीज़`

**Comment by mittalyashu** [comment]
Thank you for reviewing the changes @sayanchowdhury.

**Review by Aut0R3V — COMMENTED** [review]
I think after making some changes, this document looks great!

File: content/hi/docs/home/_index.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+    weight: 20
+    post: >
+      <p>कुबरनेट्स का उपयोग वैचारिक, ट्यूटोरियल और संदर्भ प्रलेखन के साथ करना सीखें। आप <a href="/editdocs/" data-auto-burger-exclude data-proofer-ignore>डॉक्स में योगदान</a> करने में सहायता भी कर सकते हैं!</p>
+description: >
+  कुबरनेट्स एक ओपन सोर्स container orchestration engine for automating deployment, scaling, and management of containerized applications. यह ओपन सोर्स प्रोजेक्ट को क्लाउड नेटिव कम्प्यूटिंग फाउंडेशन।
+overview: >
+  कुबरनेट्स एक ओपन सोर्स container orchestration engine for automating deployment, scaling, and management of containerized applications. यह ओपन सोर्स प्रोजेक्ट को क्लाउड नेटिव कम्प्यूटिंग फाउंडेशन (<a href="https://www.cncf.io/about">CNCF</a>) द्वारा होस्ट किया गया है।
+cards:
+- name: concepts
+  title: "मूल बातें समझें"
```
Comment (himanshu007-creator) [inline content/hi/docs/home/_index.md:25]: title: "मूल सिद्धांत समझें"

**Comment by kbhawkey** [comment]
Hello @mittalyashu .
Just checking on the status of this PR.
Are you actively working on this content?
If not, I suggest to close this PR.

**Pull request closed without merging**